standard_renderer.go

```diff
@@ -459,6 +473,13 @@ func (r *standardRenderer) setWindowTitle(title string) {
 	r.execute(ansi.SetWindowTitle(title))
 }
 
+func (r *standardRenderer) enableSynchronizedOutput() {
+	r.mtx.Lock()
+	defer r.mtx.Unlock()
+
+	r.bpActive = true
```

Comment:
Err, my dog ate my suggestion, here it is:

```suggestion
	r.soActive = true
```

Reply:
Oh god 😱 
Thx a lot @vito  !